### DISCUSSION TITLE
building a new project with react-native-windows fails

### DISCUSSION THREAD

**Discussion by Gangceen** [microsoft/react-native-windows#13417] [General]
I'm following the steps on the "getting started" documentation of react native for windows and when running "npx react-native run-windows" - this part fails with the error:

× Building Solution: C:\Program Files (x86)\Windows Kits\10\bin\10.0.19041.0\XamlCompiler\Microsoft.Windows.UI.X...
 × Build failed with message C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.32.31326\include\yvals.h(12,10): fatal  error C1083: Cannot open include file: 'crtdbg.h': No such file or directory (compiling source file Unicode.cpp) [C:\Sample projects\samplereactnative\node_modules\react-native-windows\Common\Common.vcxproj]. Check your build configuration.

Being a bit new with projects in visual studio, I do not know how to proceed, and I don't see much documentation on this. Hoping someone could point me in the right direction to tell me what I'm doing wrong.

**Comment by allisonChilton** [comment]
I am having the same issue. It appears there is some issue with case sensitivity. If i edit Common.vcxproj to change Unicode to `unicode` compilation works for that step. Other steps involving fmt.pch fail, for what I believe to be similar reasons (The pathing points to a lower case `debug` rather than Debug).

**Reply by allisonChilton** [reply]
Well, I figured out my own problem. I didn't even know Windows had this capability so I'm not sure how it got enabled but apparently you can now enable case sensitivity. This needs to be disabled in the parent directory before running the tools. See: https://learn.microsoft.com/en-us/windows/wsl/case-sensitivity

Maybe should consider adding to the documentation / environment check script

**Reply by Gangceen** [reply]
Many thanks for providing your findings to your issue, I was able to resolve mine by doing trial and error with the installed individual components, as it seems that I may have selected incorrect components on my first install. I will, however, keep the issue regarding the case-sensitivity capability in mind in case I do encounter the same.
